packages/react-label/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-label",
-  "version": "9.0.0-beta.4",
+  "version": "9.0.0-rc.0",
```

Comment:
Been discussing this in chat but for the record, the Redmond team is talking about whether to wait on putting Label in RC since none of the other inputs are yet (and there’s a very slight chance we could want API changes as we work on them more).

Reply:
#21672